client/components/RelatedProductsAndOutfits/ProductImage.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import axios from "axios";
+class ProductImage extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      image: null,
+      name: null,
+    }
+  }
+  componentDidMount() {
+    axios.get(`/api/products/${this.props.relatedProduct}/styles`)
+    .then((data) => {
+      var image = this.state.image;
+      //console.log(data)
+      //? difference between thumbnail url and url? I dont see one. What does 'default?' refers to.
+      var dataResults = data.data.results;
+      dataResults.forEach(style => {
+        if (style['default?'] === true ) {
+          image = style.photos[0].thumbnail_url;
+          //break;
+          // image is no longer null. set state.
+        }
+        if (image === null) {
+          image = style.photos[0].url;
+        }
```

Comment:
could this be outside of the for-loop?

Reply:
@jincheng411  My take is inside because it does the same deal as the previous if statement (if it is not style['default?'] It will continue to be null, and thus be set there. Can chat about it in zoom if you like.